feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -236,5 +253,8 @@ fun PreviewScreen(
                 }
             }
         }
+
+        // Screen flash overlay that stays on top of everything but invisible normally
+        ScreenFlash()
```

Comment:
Yea, it's better to add it conditionally. Refactored accordingly.

Reply:
Actually, if the condition's like whether it is screen flash mode is on, it won't work because an image capture from screen flash mode may be ongoing even after flash mode has been changed. Added a comment here in latest commits to clarify that.

Reply:
Gotcha. Seems fine to leave in.